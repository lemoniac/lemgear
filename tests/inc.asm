ld A, 3
inc A

ld BC, 1000h
dec BC

halt
