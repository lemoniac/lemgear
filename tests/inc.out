AF : 0400
BC : 0FFF
